docs/cubeviz/import_data.rst

```diff
@@ -59,6 +63,9 @@ application. A notification will appear to confirm whether the data import
 was successful. Afterward, the new data set can be found in the :guilabel:`Data`
 tab of each viewer's options menu as described in :ref:`cubeviz-selecting-data`.
 
+Due to the limitations of GUI interactions, only filenames are accepted as inputs.
+If you wish to load native Python objects, see :ref:`cubeviz-import-api`.
+
```

Comment:
What is this supposed to cover?  Has anyone tried or expected to be able to import a python object from the UI?